test/absinthe/phase/parse_test.exs

```diff
@@ -29,6 +29,34 @@ defmodule Absinthe.Phase.ParseTest do
            ] == bp.execution.validation_errors
   end
 
+  @graphql "aa;bbbbbbbb—cc"
+  test "should provide sample of parsing failure respecting unicode boundary" do
+    assert {:error, bp} = Absinthe.Phase.Parse.run(@graphql, jump_phases: false)
+
+    assert [
+             %Absinthe.Phase.Error{
+               extra: %{},
+               locations: [%{column: 3, line: 1}],
+               message: "Parsing failed at `;bbbbbbbb—`",
```

Comment:
If we're thinking character-oriented and not byte oriented I would expect 10 characters. In practice this test will fail because the message will include a sample of ten bytes.